lib/Types/Events.ts

```diff
@@ -0,0 +1,48 @@
+import { House } from "../Collections/House";
+import { Room } from "../Collections/Room";
+import { User } from "../Collections/User";
+import { Member } from "../Collections/Member";
+import { APIBaseRoom, BaseRoom } from "./Room";
+
+// `any` isnt a strict type I know but its more user friendly than casting `unknown` to a payload type they dont know
+export interface ClientEvents {
```

Comment:
Looks great! Just wondering, why are we defining the events in both upper and lower-case? 🤔

Reply:
Lower-case is usually your parsed data and upper-case is raw, though in some cases its the same, but for the sake of consistency I still added it there. A example of this is `room_create` and `ROOM_CREATE` where one is parsed and one is the raw data from the API.